src/adapter/src/coord/dataflows.rs

```diff
@@ -97,6 +97,8 @@ pub struct DataflowBuilder<'a> {
     pub compute: ComputeInstanceSnapshot,
     /// Indexes to be ignored even if they are present in the catalog.
     pub ignored_indexes: BTreeSet<GlobalId>,
+    /// Whether to eagerly build delta joins
+    pub enable_eager_delta_joins: bool,
```

Comment:
One thought, we're plumbing this `bool` around in a lot of places. It might be nice to have an enum like:

```
enum DeltaJoinStrategy {
  Eager,
  Normal,
}
```

So at any point in the code base, what the `bool` does is more descriptive

Reply:
Thanks! Skipping this for now, because (a) hopefully this flag is temporary, just for the experiment, and (b) a bunch of the plumbing should go away when we fully switch over to the new optimizer interface.